blueocean-rest-impl/src/main/webapp/scripts/analytics.js

```diff
@@ -0,0 +1,10 @@
+
+window.addEventListener('load', function() {
+    var eventData = { name: 'pageview', properties: { mode: 'classic' } };
```

Comment:
mode `classic`?

Reply:
This event will send pageviews events only when classic pages are loaded.